TickSpec.XunitWiring/FeatureSetup.fs

```diff
@@ -0,0 +1,66 @@
+﻿namespace TickSpec.XunitWiring
```

Comment:
What is the `Wiring` adding here?

Reply:
🤔 Does it make sense to call it `TickSpec.Xunit2`, or do we feel it will be a) possible b) worthwhile to have one assembly support both XUnit V2 and V3 (picture having to do ugly conditional reflection stuff if V3 has breaking changes) ?

If one knew whether `IXUnitSerializable` is being carried forward into V3, it might help one decide.

We could also defer the decision and say we'll call the V3 wiring `TickSpec.Xunit3` iff that becomes necessary

Reply:
changed back to TickSpec.Xunit